### PR TITLE
Combined photoDetailPage with photoPage

### PR DESCRIPTION
**Pull request by natalieacevedo**

### CODE REVIEW

File: src/pages/PhotoPage.tsx [hunk 1]
```diff
@@ -9,32 +9,62 @@ type PhotoPageProps = {
...
+  }, [toggleHeader, windWidth])
 
-  const { photoId } = useParams<{ photoId: string }>()
-  const { photoQuery } = useApi()
+  const { albumId, photoId } = useParams<{ albumId: string; photoId: string }>()
 
-  const { data, error, isError, isLoading } = photoQuery(photoId)
-  console.log(data)
+  const { photoInAlbum, photoQuery } = useApi()
+  if (!albumId) {
```
Comment (haplo) [inline src/pages/PhotoPage.tsx]: You can reduce the code inside the if to just the API query, the rest will be the same:

```typescript
const { data, error, isError, isLoading } = albumId
  ? photoInAlbumQuery(albumId, photoId)
  : photoQuery(photoId)

if (isLoading) {
  return <h1>Info is loading</h1>
}
// etc
```

File: src/pages/PhotoPage.tsx [hunk 2]
```diff
@@ -9,32 +9,62 @@ type PhotoPageProps = {
...
+    }
+  }, [toggleHeader, windWidth])
 
-  const { photoId } = useParams<{ photoId: string }>()
-  const { photoQuery } = useApi()
+  const { albumId, photoId } = useParams<{ albumId: string; photoId: string }>()
 
-  const { data, error, isError, isLoading } = photoQuery(photoId)
-  console.log(data)
+  const { photoInAlbum, photoQuery } = useApi()
```
Comment (haplo) [inline src/pages/PhotoPage.tsx]: Please add `Query` suffix to `photoInAlbum` to keep it in the same style as the rest.

**Review by haplo — CHANGES_REQUESTED** [review]

File: src/pages/PhotoPage.tsx (hunk 1 above)
Comment (natalieacevedo) [reply src/pages/PhotoPage.tsx]: Absolutely, this way is much more cleaner!

**Review by haplo — APPROVED** [review]
LGTM

**Pull request merged**